src/main/java/dev/tricht/lunaris/item/parser/NamePart.java

```diff
@@ -113,7 +113,7 @@ public ItemType getItemType() {
         if (name.contains("Map")) {
             return new MapItem();
         }
-        if (jewelPattern.matcher(name).matches()) {
+        if (jewelPattern.matcher(name).lookingAt()) {
```

Comment:
This change is necessary for magic items. Otherwise it would be needed to change how the item name is resolved, and that is a much bigger change. Please note that this applies to (perhaps?) every other magic item, which means that magic items are being parsed as unknown by the application. So, if hopefully there are no side effects, all these matches should be changed to `lookingAt` instead of `matches`